spec/controllers/api/v1/locations_controller_spec.rb

```diff
@@ -62,4 +61,15 @@
     expect(returned_json["location"]["rating"]).to eq(1)
     expect(returned_json["location"]["name"]).to eq( "Top of the state")
   end
+
+  it "should delete a datespot" do
+    sign_in(test_user)
+    get :destroy, params: {id: test_location.id, _json: 1}
+
+    returned_json = JSON.parse(response.body)
+    expect(response.status).to eq 200
+    expect(response.content_type).to eq("application/json")
+
+    expect(returned_json.length).to eq 0
+  end
```

Comment:
um, I realize you have had PRs merged that allowed this, but each of these it blocks should also have a describe block that denotes what CRUD action is being used..